platforms/hyperledger-indy/scripts/indy_nym_txn/pool-transaction.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+
+cd indy
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88
+sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial stable"
+sudo apt-get update
+sudo apt-get install -y libindy
+
+sudo apt-get install -y indy-cli
+
+indy-cli generate_txn.txt
```

Comment:
How would you parse the output from the get-nym? (e.g. try to get the verkey)

Reply:
using ledger get-nym did="" send=false, the created request will be printed and stored into the cli context as json object.

Reply:
send - (optional) Send the request to the Ledger (True by default). If false then created request will be printed and stored into CLI context.
That means we won't query the ledger at all...